test/unit_tests.conf

```diff
@@ -44,6 +44,10 @@ module.exports = function(config) {
     {pattern: closurePath + 'third_party/closure/goog/**/*.js', included: false},
 
     {pattern: 'static_files/images/**/*', included: false},
+
+    {pattern: 'genfiles/core/css/**/*', included: false},
```

Comment:
The units tests are not creating any genfiles at all, so if your test needs genfiles it should be moved to another test like core_tests.